Dockerfile.dapper718299099

```diff
@@ -0,0 +1,10 @@
+FROM quay.io/submariner/shipyard-dapper-base:devel
```

Comment:
This file needs to be removed.

Reply:
Argh, forgot to cleanup before updating. Will do so when I rebase.